layouts/partials/community_links.html

```diff
@@ -2,19 +2,19 @@
 
 <section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
 <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
-<h2>Learn and Connect</h2>		
-<p>Using or want to use {{ .Site.Title }}? Find out more here:
+<h2>{{ T "community_learn" }}</h2>
+<p>{{ T "community_using" }} {{ .Site.Title }}? {{ T "community_find_out" }}:
 {{ with index $links "user"}}
 {{ template "community-links-list"  . }}
 {{ end }}
 </div>
 <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
-<h2>Develop and Contribute</h2>
-<p>If you want to get more involved by contributing to {{ .Site.Title }}, join us here:
+<h2>{{ T "community_develop" }}</h2>
+<p>{{ T "community_contribute" }} {{ .Site.Title }}, {{ T "community_join_us" }}:
```

Comment:
I don't like the way translations are implemented here: `.Site.Title` is always at the end of the sentence, which is true for English, but may not be true for other languages. I think we should pass the `.` context when calling T here. I would like to see something like:

```
<p>{{ T "community_contribute" . }}, {{ T "community_join_us" }}:
``` 

Inside my de.toml, I could then use:

```
[community_contribute]
other = "Wenn Du aktiv am {{ .Site.Title }}-Projekt beteiligen und einen Beitrag dazu leisten möchtest"
```